### PR TITLE
Stable diffusion section

### LINKED ISSUE #13: Write Stable Diffusion section
Write the Section on Stable Diffusion generator

### PR DESCRIPTION
**Pull request by EhrenbruderRoschtbraten**
Wrote down the section about the Stable Diffusion model.

closes #13

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -62,6 +62,9 @@ \section{Related work}
 \subsection{Generating models}
 
 \subsubsection{Image generating models}
+The first image generating model we us is the Stable Diffusion model. It is developed by engineers and researchers from CompVis (\url{https://github.com/CompVis}), Stability AI (\url{https://stability.ai/}) and LAION (\url{https://laion.ai/}) \cite{patil2022stable}.
```
Comment (RaphaW) [inline paper.tex]:
```suggestion
The first image generative model we use is the Stable Diffusion model. It is developed by engineers and researchers from CompVis (\url{https://github.com/CompVis}), Stability AI (\url{https://stability.ai/}) and LAION (\url{https://laion.ai/}) \cite{patil2022stable}. 
```

**Review by RaphaW — APPROVED** [review]
YOu some small changes

**Review by 1Maxnet1 — COMMENTED** [review]
In the sentence were we mention that we used Stable Diffusion I would mention which version

**Review by 1Maxnet1 — APPROVED** [review]
LGTM

**Pull request merged**